packages/block-library/src/site-logo/editor.scss

```diff
@@ -47,24 +44,27 @@
 // @todo this particular minimal style of placeholder could be componentized further.
 .wp-block-site-logo.wp-block-site-logo {
 
+	/**
 	&.is-default-size .components-placeholder {
 		height: 60px;
 		width: 60px;
 	}
```

Comment:
If I don't remove the height and width, the upload button i misaligned when there is a border radius.

Reply:
I am not sure what this CSS is for, was 60px chosen because the default size of the logo is 120?

Reply:
Ok, I found the ticket https://github.com/WordPress/gutenberg/pull/48218
I am not sure how to resolve it without letting it fall back to `height: 100% `and `width:100%` though! @richtabor

Reply:
> I am not sure what this CSS is for, was 60px chosen because the default size of the logo is 120?

Yes, per https://github.com/WordPress/gutenberg/pull/48218—120x120 is too big for a logo placeholder, but 120px is the max initial width for when a logo is uploaded.

Reply:
This is a tricky one 🤔 

While in the placeholder state the layout shift when adding a border sticks out. When not in the placeholder state the block has `border-box` set for the `box-sizing`. Adding `border-box` for the placeholder state exacerbates the problem with scrollbars etc.

I'm not sure there is a bulletproof solution but maybe there is something we can do to smooth out the experience a bit for the less extreme use cases.

If we avoid applying the custom border on the placeholder directly and instead apply it to a new pseudo element. We can position the bordered pseudo element over the inner contents. It will start to crop the underlying button as the border increases but that seems better than scrollbars.

Hacking around in dev tools this is what I got:


https://github.com/user-attachments/assets/18831190-918b-4b51-ac0e-98c0f484e21f